contracts/contracts/PreConfCommitmentStore.sol

```diff
@@ -269,9 +274,9 @@ contract PreConfCommitmentStore is IPreConfCommitmentStore, Ownable2StepUpgradea
         commitmentIndex = getCommitmentIndex(newCommitment);
 
         // Store the new commitment
-        commitments[commitmentIndex] = newCommitment;
-        // Mark the encrypted commitment as used
-        encryptedCommitment.isUsed = true;
+        openedCommitments[commitmentIndex] = newCommitment;
+        // Mark the unopened commitment as used
+        unopenedCommitment.isUsed = true;
```

Comment:
`isUsed` really means `isOpened` right? Might be good to change the name of that boolean but not super important
```suggestion
        unopenedCommitment.isUsed = true;
```